carbon-design-system/carbon
# fix(multiselect): remove aria-selected prop

ref #6411 

Downshift follows (the sort of weird) [WAI-ARIA spec for listboxes](https://www.w3.org/TR/wai-aria-practices/examples/combobox/aria1.1pattern/listbox-combo.html) and sets `aria-selected="true"` for any highlighted items. We were setting this ourselves and breaking this functionality. This removes us setting that prop and let's Downshift handle this for us :+1: 

### to test
Open up Multiselect, pop open the listbox with the dev tools inspector open and see the `aria-selected` value change from `true` to `false` on mouse hover for any list item
